### DISCUSSION TITLE
Adding Gas Use Data

### DISCUSSION THREAD

**Discussion by ccsnet** [community/community#65787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, just found your dash recently and liked it that much I have added to my homeassistant dash as a tab.

Although this one is about energy generation I was wondering if you would consider putting live gas use on as well?

I found uk data and feeds (I think by request but there are some csv and xls options too) here?

https://data.nationalgas.com/

Thanks

Terran

**Comment by Akash1134** [comment]
Hi @ccsnet ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
